developer-docs/README.md

```diff
@@ -1,48 +1,49 @@
 
-Some [recommended resources for Docker and Kubernetes are available here](./recommended-reading.md)
+For further information on Docker and Kubernetes:
+* Docker
+  * [Docker getting started tutorial](https://docs.docker.com/engine/getstarted/)  
+* Kubernetes
+  * [What is Kubernetes?](http://kubernetes.io/docs/whatisk8s/)  
+  * [Kubernetes talk to introduce the concepts](https://www.youtube.com/watch?v=5gz8kOUstFc)
 
-## Projects and namespaces
-Kubernetes resources (including your applications!) are always deployed into a particular namespace.
-These namespaces provide separation between the different projects hosted on the platform.
-We also recommend that each project has one namespace per environment (e.g. for dev, preprod, prod)
-
-## Project getting started guide
-For new projects you will need to put a PR into the hosting platform to create new namespaces.
-Instructions are available here:  
-https://gitlab.digital.homeoffice.gov.uk/Devops/kube-hod-platform
+## Developer getting started guide
+Some prerequisites are required before developing on ACP. [This guide](dev_setup.md) will show you how to:
 
-We recommend you have one per environment, with each namespace prefixed with the project name.
-For example if the project name were *boomba* the namespaces might be *boomba-dev*, *boomba-uat*, *boomba-preprod*.
+  * Get access to the VPNs that allows you to connect to the platform
+  * Get setup with the kubectl client that lets you deploy applications to kubernetes
+  * Get access to Quay and Artifactory
 
-You will also need to ask for your team to be granted access to the new namespaces, which you can do by making a request here:  
-https://github.com/UKHomeOffice/application-container-platform-bau/projects/1
+In addition, new developers should look at the [new user flow](../docs/newuser.md) documentation whilst going through this doc to serve as a checklist and make sure you are onboarded to all necessary platform services.
 
-## Project getting to production guide
-Content to follow soon! In the meantime please ask for help in the Devops channel on the [HOD DSP Slack](https://hod-dsp.slack.com)
+## Platform Hub
+  [The Platform Hub](https://hub.acp.homeoffice.gov.uk) serves as a central portal for users of ACP. It acts as an all-in-one place to find information, requests and also support for the platform. The hub also provides tools to develop, build, deploy and manage all your projects.
 
-## Developer getting started guide
-[This guide](dev_setup.md) shows you how to:
+  Support requests within the hub are located [here](https://hub.acp.homeoffice.gov.uk/help/support/requests/overview) or within the sidebar of the Platform hub. These request templates cover a wide majority of general requests that users normally need. If your issue/request is not listed, please get in touch with us using the [Contact Us](https://hub.acp.homeoffice.gov.uk/help/support/requests/new/contact-us) support request.
```

Comment:
So i've put an explanation for support requests here instead of `Project getting started guide`. I think it flows better - rather than reading up on the platform hub and then have one tiny paragraph about support requests further down the doc.
What do you guys think?

Reply:
Sounds like a good plan.

Suggestion: it may be nicer for the user if the first sentence started with something like:

```md
You can submit [support requests via the hub](https://hub.acp.homeoffice.gov.uk/help/support/requests/overview)
```